definitions/pipelines/rnaseq_star_fusion.cwl

```diff
@@ -98,6 +103,20 @@ steps:
             min_readlength: trimming_min_readlength
         out:
             [fastqs, fastq1, fastq2]
+    check_strand:
+        run: ../tools/check_strand.cwl
+        in:
+            gtf_file: gtf_file
+            kallisto_index: kallisto_index
+            cdna_fasta: cdna_fasta
+            reads1:
+                source: bam_to_trimmed_fastq/fastq1
+                linkMerge: merge_flattened
+            reads2:
+                source: bam_to_trimmed_fastq/fastq2
+                linkMerge: merge_flattened
```

Comment:
I believe this step will need to scatter on reads1 and reads2 to account for handling multiple instrument data in a single workflow run.